make_gk2_mswin_installer.cmd

```diff
@@ -1,18 +1,61 @@
+@echo off
+
+set GK_VER=2.16.2
+
+@echo/
+@echo ***************************************
+@echo ** Create GEDKeeper setup executable **
+@echo ***************************************
+@echo/
+
+set CONFIG_TYPE=Debug
+for %%a in (release Release RELEASE) do if (%%a)==(%1) SET CONFIG_TYPE=Release
+
+if not exist .\clean.cmd goto no_clean
+if not exist .\clean_all.cmd goto no_clean
+if not exist .\build_all.mswin.x86.cmd goto no_builder
+if not exist "C:\Program Files (x86)\NSIS\makensis.exe" goto no_nsis
+if not exist "c:\Program Files\7-zip\7z.exe" goto no_7zip

```

Comment:
#246 uses `%PROGRAMFILES%\7-zip\7z.exe`[1] when it refers 7z.  May be we should use it here too?

[1] https://github.com/Serg-Norseman/GEDKeeper/blob/6b07575b397d5bebbd79e74d51cd3f37e8e19757/deploy/gk2_win_portable.cmd#L12